### PR TITLE
wip: tests compilation fixes

### PR DESCRIPTION
**Pull request by mmoadeli**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/routing/473)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @inetic

(maidsafe_highfive has picked a reviewer for you, use r? to override)

**Comment by mmoadeli** [comment]
this is WIP

**Comment by benjaminbollen** [comment]
@inetic can we "quick review" and merge this?

File: src/routing_membrane.rs [hunk 1]
```diff
@@ -1558,57 +1536,99 @@ fn populate_routing_node() -> RoutingMembrane<TestInterface> {
...
+                   Authority::Unknown);
         assert_eq!(result_stats.call_count, 1usize);
         assert_eq!(result_stats.data, "UnauthorisedPut".to_string().into_bytes());
     }
 
 #[test]
     fn call_handle_put_response() {
-        let put_data_response: PutDataResponse = Random::generate_random();
-        assert_eq!(call_operation(put_data_response, MessageType::PutDataResponse,
-             Arc::new(Mutex::new(Stats::new())), Authority::NaeManager(Random::generate_random()), None, None).call_count, 1usize);
```
Comment (benjaminbollen) [inline src/routing_membrane.rs]: prefer using #[ignore] in the future, keeps better track of commented-out tests cheers

**Comment by benjaminbollen** [comment]
looks good to me

**Pull request merged**